docs/modules/build-system/build-config.md

```diff
@@ -0,0 +1,47 @@
+# Build configuration
+
+The Build System feature module uses a scriptable object for its build configuration. 
```

Comment:
should feature module be capitalized here? ;)

Reply:
To @ElleEss's sadness, yes